### PR TITLE
Add descriptions to SDG view

### PR DESCRIPTION
**Pull request by javierm**
## Objectives

* Add a long description to SDGs, using official UN texts
* Add a long description with links to read more/ read less

## Visual Changes
- read more
Captura de pantalla 2021-01-26 a las 18 28 50

- read less
Captura de pantalla 2021-01-26 a las 18 29 00

### CODE REVIEW

File: app/assets/stylesheets/sdg/goals/show.scss [hunk 1]
```diff
@@ -63,6 +63,21 @@
         }
       }
     }
+
+    .goal-description-content {
+
+      .goal-description {
+
+        &.short {
```
Comment (javierm) [inline app/assets/stylesheets/sdg/goals/show.scss:71]: SCSS Lint is giving us a warning: Nesting should be no greater than 4, but was 5.

One option here:

```scss
    .goal-description-content {

      .read-more {
        text-align: center;
      }
    }

    .goal-description {

      &.short {
        height: rem-calc(150);
        overflow: hidden;
      }
    }
```

Another option:

```scss
    .goal-description {

      &.short {
        height: rem-calc(150);
        overflow: hidden;
      }

      + .read-more {
        text-align: center;
      }
    }
```

File: spec/components/sdg/goals/show_component_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    expect(page).to have_content "No Poverty"
+  end
+
+  it "renders a long description" do
+    component = SDG::Goals::ShowComponent.new(goal_1)
+
+    render_inline component
+
+    expect(page).to have_css "#description_goal_#{goal_1.code}"
+    expect(page).to have_content "Globally, the number of people living in extreme poverty declined from 36 per cent in 1990 to 10 "
```
Comment (javierm) [inline spec/components/sdg/goals/show_component_spec.rb:25]: Maybe a bit less information is enough :smiling_imp:.

```suggestion
    expect(page).to have_content "Globally, the number of people living in extreme poverty"
```

**Review by javierm — COMMENTED** [review]
Great! Just a couple of lint warnings we were getting.

**Review by javierm — COMMENTED** [review]
Looks great!

**Review by taitus — APPROVED** [review]

**Pull request merged**